README.md

```diff
@@ -21,6 +21,15 @@ regular Homebrew package. For example:
 
 Check out the formulas in this repository to see what is available.
 
+## Beta Releases
+
+To install beta channel releases, instead of the stable ones, add a `--head`
+flag after the brew commands:
+
+```shell
+brew install jamf-pro --head
```

Comment:
It would be nice to use --HEAD here to match the capitalization the homebrew itself uses in its documentation. https://docs.brew.sh/Manpage#fetch-options-formula

Reply:
Good catch, I've changed it.